.github/workflows/builds.yml

```diff
@@ -2589,8 +2590,8 @@ jobs:
       - run: tar xvf prep/libhtp.tar.gz
       - run: tar xvf prep/suricata-update.tar.gz
       - run: ./autogen.sh
-      - run: CFLAGS="${DEFAULT_CFLAGS}" ./configure --enable-unittests
-      - run: make -j ${{ env.CPUS }}
```

Comment:
@jasonish I see that you committed this with 8522256aaa1

But env.CPUs does not seem to work here...

Reply:
Yup, that was a mistake.

Reply:
Is there more to fix ?